content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+1. In the **Class to create** dialog box, select `InputEventMap`, and choose **OK**.
+
+1. Specify changes for the Event Generator. Each Event Generator has a set of properties that you can customize.
+
+### Setting Event Generator properties
+
+The following `.inputbindings` file specifies a keyboard for the device type and the **Spacebar** for the input **Event Name**.
+
+   ![Input bindings configuration example in Asset Editor.](/images/user-guide/interactivity/input/inputbindings-example.png)
+
+### Saving your input bindings asset
+
+1. In the **Asset Editor**, choose **File, Save**.
+
+1. Enter a name for the `.inputbindings` file and choose **Save**.
+
+1. In Entity Inspector, in the Input component, for **Input to event bindings**, click the **Browse Directory** icon and select your `.inputbindings` file.
+
+## Mapping Input Events to gameplay logic
+
+After you create an `.inputbindings` file and specify Input Events, you can use Script Canvas or Lua to map the Input Events to gameplay logic. You can create gameplay logic in a visual scripting environment with **Script Canvas Editor** or write your own Lua scripts with **Lua Editor (Lua IDE)**.
+
+### Using input in Script Canvas 
+
+You can create a Script Canvas graph that connects to Input Events. For more information about scripting with Script Canvas, refer to [Creating Gameplay with Script Canvas](/docs/user-guide/scripting/script-canvas).
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component** and add the **Script Canvas** component.
+
+1. In the Script Canvas component, for **Script Canvas Asset**, click the **Open in Script Canvas Editor** icon and create a new Script Canvas graph like the following.
+
+### Example Script Canvas graph
+
+In the following graph, the **Input Handler** node connects `Action` events to various **Transform** nodes.  The **Pressed**, **Held**, and **Released** outputs are triggered when the state of an Event Generator changes.
+
+![Example Script Canvas graph for the Input component](/images/user-guide/interactivity/input/sc-input-example.png)
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to Input Events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, click the **Open in Lua Editor** icon.
+
+1. In the Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
```

Comment:
```suggestion
1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the input event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
```
Review the wording of this step. As currently written, I can't make sense of it. (There might be some stray prepositions adding to my confusion.) If possible, break it into two sentences for additional clarity.